README.md

```diff
@@ -3,7 +3,7 @@
 > Account client API for the browser
 
 [![Build Status](https://travis-ci.org/hoodiehq/hoodie-account-client.svg?branch=master)](https://travis-ci.org/hoodiehq/hoodie-account-client)
-[![Coverage Status](https://coveralls.io/repos/hoodiehq/hoodie-account-client/badge.svg?branch=master)](https://coveralls.io/r/hoodiehq/hoodie-account-client?branch=master)
+[![Coverage Status](https://coveralls.io/github/hoodiehq/hoodie-account-client?branch=master)](https://coveralls.io/github/hoodiehq/hoodie-account-client?branch=master)
```

Comment:
Please leave the first URL as it was, it’s the URL for the image: https://coveralls.io/repos/hoodiehq/hoodie-account-client/badge.svg?branch=master

Only change the 2nd URL (as you already did correctly :)